### PR TITLE
devtools-modules: bump version to 0.0.29

### PR DESCRIPTION
**Pull request by juliandescottes**
Nicolas, do you know if there is any process to release a new devtools-modules version?

### CODE REVIEW

**Comment by nchevobbe** [comment]
I don't know at all :/ 
The last PR for this looks like that https://github.com/devtools-html/devtools-core/commit/91f5b7f98362a4ee3870f5fb1d78ff56d519a9c0 , I don't know if there are additional things to do.

**Comment by juliandescottes** [comment]
ok let's ping @jasonLaster :) 

(I already released it on npm though, because I needed it for a fix in mozilla-central, sorry :P)

**Comment by juliandescottes** [comment]
Thanks for having a look! 

Just bumped launchpad and context menu to use v0.0.29, switching r? to @jasonLaster

File: packages/devtools-launchpad/package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "devtools-config": "^0.0.15",
     "devtools-connection": "^0.0.6",
     "devtools-contextmenu": "^0.0.3",
-    "devtools-modules": "^0.0.28",
```
Comment (jasonLaster) [inline packages/devtools-launchpad/package.json]: i think you should also do `yarn install` and bump the lockfiles

Comment (juliandescottes) [inline packages/devtools-launchpad/package.json]: done!

**Comment by juliandescottes** [comment]
Totally forgot to merge this.

**Pull request merged**